docs/current_docs/api/module-tests.mdx

```diff
@@ -0,0 +1,80 @@
+---
+slug: /api/module-tests
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import DaggerModuleInit from '../partials/_dagger_module_init.mdx';
+
+# Module Tests
+
+Like any other piece of software, Dagger Functions and modules should be thoroughly tested. This section documents proven patterns and best practices for effectively testing reusable modules.
+
+## Test module
```

Comment:
From the text below, it seems like there are two patterns: test module and testable examples. If my understanding is correct, should we restructure these subsections slightly so the two patterns are described first:
- Test module pattern
- Testable example module pattern
- Test function signature
- All function

Reply:
The testable example pattern is an "extension" of the test module pattern.

Restructuring is fine by me.